packages/forms-web-app/src/controllers/examination/email.js

```diff
@@ -2,24 +2,61 @@ const {
 	routesConfig: {
 		examination: {
 			directory: examinationDirectory,
-			pages: { nameMyself, email }
+			pages: { email }
 		}
 	}
 } = require('../../routes/config');
 
+const config = require('../../config');
+const examinationSessionStorage = config?.sessionStorage?.examination;
+
 const pageData = {
-	backLinkUrl: `${examinationDirectory + nameMyself.route}`,
 	id: email.id,
 	pageTitle: email.name,
-	title: email.name
+	title: email.name,
+	hint: "We'll use your email address to confirm we've received your submission. We will not publish your email address."
 };
 
 const getEmail = async (req, res) => {
+	const requestSession = req.session;
+	const userEmail = requestSession?.[examinationSessionStorage.name]?.userEmail;
+	const currentView = requestSession?.currentView;
+	userEmail && (pageData.email = userEmail);
+
+	if (!requestSession || !currentView || !currentView?.route)
```

Comment:
We would always have a req.session so probably no need to check for it.

Reply:
it breaks the tests if you forget to pass it and in case someone gets to the link in an anonymous tab they should get a 404

Reply:
As mentioned below, the user would always have an req.session as it is applied to Express.js. We want to check for req.session.examination